packages/safe-tools-client/tests/acceptance/wallet-connection-test.ts

```diff
@@ -162,16 +167,22 @@ module('Acceptance | wallet connection', function (hooks) {
   module('Remembering the selected chain', function () {
     test('Defaults to mainnet', async function (assert) {
       await visit('/schedule');
-      assert.dom('[data-test-selected-network]').hasText('Ethereum Mainnet');
+      await click('[data-test-connect-wallet-button-modal-sidebar]');
```

Comment:
These 2 tests needed a fix because we don't display the network dropdown anymore when the wallet is not connected.